azkaban-common/src/main/java/azkaban/project/AzkabanProjectLoader.java

```diff
@@ -209,8 +212,13 @@ private void persistProject(final Project project, final FlowLoader loader, fina
         this.projectLoader.uploadProjectProperties(project, directoryFlowLoader.getPropsList());
 
       } else if (loader instanceof DirectoryYamlFlowLoader) {
-        // Todo jamiesjc: upload yaml file to DB as a blob
-
+        final File[] flowFiles = projectDir.listFiles(new SuffixFilter(Constants.FLOW_FILE_SUFFIX));
+        for (final File file : flowFiles) {
```

Comment:
one line?
for (final File file : projectDir.listFiles(new SuffixFilter(Constants.FLOW_FILE_SUFFIX)))

Reply:
Will make the change together with next PR.